appcli/commands/configure_template_cli.py

```diff
@@ -71,10 +67,10 @@ def ls(ctx):
         @click.argument("template_rel_path")
         @click.pass_context
         def get(ctx, template_rel_path):
```

Comment:
We need to rename this parameter, `template_rel_path` is a terrible name, especially as its exposed to the end user via click
I think just go with `template_path`.

Reply:
`--template`
Check verbage around 'baseline'